includes/Order/functions.php

```diff
@@ -895,9 +890,8 @@ function dokan_order_csv_export( $orders, $file = null ) {
 
     fputcsv( $output, $headers );
 
-    foreach ( $orders as $order ) {
-        $line      = array();
-        $the_order = dokan()->order->get( $order->order_id );
```

Comment:
any particular reason to remove this line? reference of this variable has been used later.